vocabularies/result-type.ttl

```diff
@@ -90,144 +90,176 @@
 rslt:area a skos:Concept ;
     skos:definition "The quantity that expresses the extent of a two-dimensional figure or shape or planar lamina, in the plane. Surface area is its analog on the two-dimensional surface of a three-dimensional object."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rara" ;
     skos:prefLabel "Area"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:capacitance a skos:Concept ;
     skos:definition "The ratio of the change in electric charge of a system, to the corresponding change in its electric potential."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rcap" ;
     skos:prefLabel "Capacitance"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:casing-weight a skos:Concept ;
     skos:definition "The mass per unit of length of a length of tubular installed in a borehole."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rcsw" ;
     skos:prefLabel "Casing Weight"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:colour a skos:Concept ;
     skos:definition "The qualitative description of the colour of an object."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rcol" ;
     skos:prefLabel "Colour"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:composition a skos:Concept ;
     skos:definition "A description of the component parts of a sample or object. Typically given as a component with its percentage or ratio within a bulk sample."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
     skos:prefLabel "Composition"@en ;
+    skos:notation "rcmp" ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:compressibility a skos:Concept ;
     skos:definition "A measure of the relative volume change of a fluid or solid as a response to a pressure (or mean stress) change."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rcpb" ;
     skos:prefLabel "Compressibility"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:compressive-strength a skos:Concept ;
     skos:definition "The capacity to withstand axially directed pushing forces"@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rcps" ;
     skos:prefLabel "Compressive Strength"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:concentration a skos:Concept ;
     skos:definition "The abundance of a constituent divided by the total volume of a mixture."@en ;
     skos:inScheme <http://linked.data.gov.au/def/result-type> ;
+    skos:notation "rctn" ;
     skos:prefLabel "Concentration"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
 
 rslt:conductivity a skos:Concept ;

```

Comment:
Potential (ha) for the introduction of Thermal Conductivity.

Conductivity would become:

```rslt:electrical-conductivity a skos:Concept ;



    skos:altLabel "Electrical Conductance"@en ;



    skos:definition "A measure of a material's ability to conduct an electric current."@en ;



    skos:inScheme <http://linked.data.gov.au/def/result-type> ;



    skos:notation "recd" ;



    skos:prefLabel "Electrical Conductivity"@en ;



    skos:topConceptOf <http://linked.data.gov.au/def/result-type> .
```

and the new entry for Thermal Conductivity would be:

```rslt:thermal-conductivity a skos:Concept ;



    skos:altLabel "Thermal Conductance"@en ;



    skos:definition "A measure of a material's ability to conduct an thermal energy."@en ;



    skos:inScheme <http://linked.data.gov.au/def/result-type> ;



    skos:notation "rtcd" ;



    skos:prefLabel "Thermal Conductivity"@en ;



    skos:topConceptOf <http://linked.data.gov.au/def/result-type> .

Reply:
@KellyVance

Reply:
done